docs/ibm-cloud-rules.md

```diff
@@ -4048,7 +4048,7 @@ n/a
 <td valign=top><b>Description:</b></td>
 <td>This rule verifies that each parameter name complies with the casing convention associated with that parameter's type.
 For example, the default casing convention for query parameters is snake-case (e.g. `my-query-param`), the default for path parameters
```

Comment:
```suggestion
For example, the default casing convention for query parameters is snake-case (e.g. <code>my_query_param</code>), the default for path parameters
```
I realize you didn't change this, but my-query-param isn't snake case, is it? :)